install_config/storage_examples/azure_blob_docker_registry_example.adoc

```diff
@@ -45,7 +45,7 @@ To create a new registry pod and replace the old pod automatically:
       container: azureblobname
```

Comment:
should add `realm: core.windows.net`  in this field, not like below

Reply:
@dongboyan77 Sorry, I'm not sure if I understand. I've done that on line 48 (below). Should I be placing this somewhere else as well? Should I place it underneath the `container` line above?

Reply:
yes, please place it underneath the `container` line above, do not change anything on line 48

Reply:
Thanks @dongboyan77 . I've changed as suggested. I'll update the BZ. Please let me know if there's anything else for this PR.